test/frisby_mock_request.js

```diff
@@ -1437,4 +1485,87 @@ describe('Frisby matchers', function() {
     })
   })
 
+  describe('header checks should ignore case on field names and match case for values', function(){
```

Comment:
Given what you wrote in https://github.com/MarkHerhold/IcedFrisby/pull/73#discussion_r143760491, this should be **ignore case on field names and values** or similar.